src/handlers/user-identity/tests/user-identity-handler.test.ts

```diff
@@ -464,19 +459,7 @@ describe("user-identity-handler expired", () => {
   const RANDOM_NBF: string = "2023-04-25T15:01:36.000Z";
 
   beforeEach(() => {
-    jest.spyOn(storedIdentityValidator, "validateStoredIdentityCredentials").mockReturnValue(true);
-    jest.clearAllMocks();
-    newEvent = event();
-    jest.spyOn(configuration, "getServiceApiKey").mockResolvedValue("an-api-key");
-    jest.spyOn(configuration, "getConfiguration").mockResolvedValue({
-      evcsApiUrl: "https://evcs.gov.uk",
-      controllerAllowList: [ALLOWED_CONTROLLER],
-      fraudIssuer: [FRAUD_ISSUER],
-      fraudValidityPeriod: TEST_FRAUD_VALIDITY_HOURS,
-    } as Configuration);
     jest.spyOn(fraudCheckService, "hasFraudCheckExpired").mockRestore();
-
-    jest.useFakeTimers();
```

Comment:
Oh just curious, do we need to keep `useFakeTimers` if we use `setSystemTime`? I mean, I guess not, as everything is probably passing, but just checking this is intentional.

Reply:
Thank you for explaining.